target/scripts/start-mailserver.sh

```diff
@@ -183,22 +60,43 @@ function _register_functions
 
   # ? >> Setup
 
-  _register_setup_function '_setup_default_vars'
+  _register_setup_function '_environment_variables_export'
```

Comment:
```suggestion
```

You've got this twice, the 2nd time is further down after `_register_setup_function '_misc_save_states'`.

Since ENV does get changed further down with the `ACCOUNT_PROVISIONER` handling and `ENABLE_SASLAUTHD` condition directly after that, I assume you want to keep the 2nd call and remove this one.

Reply:
Yet another error in my thinking. I will remove the first call.

Reply:
Done in 320c45dbc20fc8e01490bb646bfb81331c5dc33e